cmake/OpenCVDetectInferenceEngine.cmake

```diff
@@ -134,10 +134,14 @@ endif()
 # Add more features to the target
 
 if(INF_ENGINE_TARGET)
+  if(InferenceEngine_VERSION VERSION_GREATER_EQUAL "2021.4")
+    math(EXPR INF_ENGINE_RELEASE "${InferenceEngine_VERSION_MAJOR} * 1000000 + ${InferenceEngine_VERSION_MINOR} * 10000 + ${InferenceEngine_VERSION_PATCH} * 100")
+  endif()
```

Comment:
BTW, It makes sense to emit warning if there is passed wrong value through CMake's command-line (need to think how to ignore/handle "patch" version here in this comparison).